eng/Tools.props

```diff
@@ -81,14 +82,16 @@
     <!-- Uninstall the global tool if it exists with a different version. -->
     <Exec Condition="Exists($(GlobalToolsDir)) AND $(DotNetListToolsOutput.Contains('%(RepoTool.Identity)')) AND !$([System.Text.RegularExpressions.Regex]::IsMatch('$(DotNetListToolsOutput)', '$([System.Text.RegularExpressions.Regex]::Escape('%(RepoTool.Identity)'))\s+$([System.Text.RegularExpressions.Regex]::Escape('%(RepoTool.Version)'))'))"
           Command="$(DotNetCmd) tool uninstall --tool-path $(GlobalToolsDir) %(RepoTool.Identity)"
+          WorkingDirectory="$(RepoRoot)"
           ContinueOnError="WarnAndContinue" />
 
     <!--
       Installs the global tool if it doesn't exist with the right version.
       Creates the global tools folder automatically if it doesn't exist.
     -->
     <Exec Condition="!Exists($(GlobalToolsDir)) OR !$([System.Text.RegularExpressions.Regex]::IsMatch('$(DotNetListToolsOutput)', '$([System.Text.RegularExpressions.Regex]::Escape('%(RepoTool.Identity)'))\s+$([System.Text.RegularExpressions.Regex]::Escape('%(RepoTool.Version)'))'))"
-          Command="$(DotNetCmd) tool install --tool-path $(GlobalToolsDir) %(RepoTool.Identity) --version %(RepoTool.Version)"
+          Command="$(DotNetCmd) tool install --tool-path $(GlobalToolsDir) %(RepoTool.Identity) --version %(RepoTool.Version) --add-source https:%2F%2Fapi.nuget.org/v3/index.json"
```

Comment:
Just add the source at the beginning of restore sources: https://github.com/dotnet/corefx/pull/33860/files#diff-c9ed7fc6ec49880569e6c2eb08120fc4R4

Reply:
That does not work for the out-of-proc Exec task.

Reply:
Right, I considered that but as there are right now some issues with the correct encoding of the sources, i.e. there are issues with semicolons and the slashes I didn't do that now. We can revisit that when we need to add global tools that aren't available on nuget.

Reply:
Ok cool.